locals.tf

```diff
@@ -2,7 +2,8 @@
 /*                                  Generics                                  */
 /* -------------------------------------------------------------------------- */
 locals {
-  cluster_name = "${var.prefix}-${var.environment}-${var.name}"
+  cluster_name_tmp = "${var.prefix}-${var.environment}-${var.name}"
+  cluster_name = substr("${local.cluster_name_tmp}", 0, min(19, length(local.cluster_name_tmp)))
```

Comment:
I'm not sure this use overall resource -> only alb (I'm nor sure) need strip right, 
other resource seem ok

Reply:
yes, I test with the long name and only alb face this problem at the moment. (limit to 32). We must to strip to 19 to concat another string.